### ISSUE TITLE
Reply to identity

### ISSUE THREAD

**Issue by geotheory** [jonbakerfish/TweetScraper#44]
Great utility.  Is it possible to collect the identity of who replies are replying to?

**Comment by jonbakerfish** [comment]
Yes. You need to get the content of _https://twitter.com/[user_id]/status/[tweet_id]_ and extract the data you want.

**Comment by geotheory** [comment]
Sorry for slow reply. I get that this is the current method. As a feature request I wonder if it might at some point be grabbed along with the rest of the tweet? Reducing the number of calls is presumably desirable.
